app/views/comments/_commentable.html.erb

```diff
@@ -79,7 +79,11 @@
   <%= flash_div :comment_error, :comment_notice %>
 
   <% commentable_parent = find_parent(commentable) %>
-  <% if AdminSetting.current.guest_comments_off? && guest? %>
+  <% if logged_in_as_admin? %>
```

Comment:
Instead of moving this above the other checks, do you think it would make sense to adjust the anon commenting checks to check for admin login instead? Specifically I am thinking that the two `!logged_in?` in this elsif chain could be `guest?` instead, because that also checks the admin login.

The reasoning for my idea is that I imagine that admins would like to be able to see the effect of the comment setting on the work without always getting the "please log out" message. But this is just an idea, I am not sure what the best solution is.

Reply:
After sleeping on it, I would be in favor of a quick `Scope: Tests Only` PR right now to fix the build, and then an addendum to AO3-5860 or a new ticket altogether to rework the behavior (if need be).